### DISCUSSION TITLE
Policy for reopening issues

### DISCUSSION THREAD

**Discussion by jtgreen-cse** [Security-Onion-Solutions/securityonion#2858] [General]
I have 2 issues that seem to have come back (or at least never fully went away). Should we open new issues or comment on the original issue to try and get it reopened.

Issues in question are #2293 and #1821.

Edit:

I've added more details and a potential fix on what I've found on #2293.

**Comment by weslambert** [comment]
HI @jtgreen-cse , In this case, you can open a new issue, and reference the previous.
